content/chainguard/chainguard-enforce/reference/data-collection.md

```diff
@@ -0,0 +1,93 @@
+---
+title : "Data Collected by Chainguard Enforce"
+linktitle: "Data Collection"
+lead: ""
+description: "An overview of the data collected by Chainguard Enforce"
+type: "article"
+date: 2023-04-17T08:48:45+00:00
+lastmod: 2023-04-17T08:48:45+00:00
+draft: false
+images: []
+weight: 025
+---
+
+At Chainguard, we have a shared awareness of data collection and retention. This document focuses on cataloging the data captured by Chainguard Enforce. It also includes a brief Frequently Asked Questions section outlining some of Chainguard's data collection practices.
+
+## Customer Data Collected by Chainguard Enforce
+
+The following subsections outline the various types of data that are collected by Chainguard Enforce, grouped by category.
+
+### IAM Data (user provided)
+
+Data regarding tenant IAM structures:
+* IAM group hierarchies
+* IAM role bindings
+
+Auxiliary metadata about a customer includes:
+* User email: we collect user emails in order to allow showing ACL audit information through `chainctl iam rolebindings list`
+
+
+### Policy Data (user provided)
+
+Information to enforce user policy, including:
+* IAM group
+* Container Image globs
+* Signing key identifying information:
+    * OIDC issuer + subject pairs
+    * URL of Fulcio server
+    * KMS key reference
+* Event subscriptions to receive compliance-related events 
+
+### Enforce Cluster Data
+
+Metadata regarding tenant clusters, including:
+* Chainguard IDs (cluster ID and cluster remote ID)
+* IAM group the cluster belongs to
+* Identity Issuer of the cluster
+* Kubernetes version
+* Chainguard Agent version
+
+Chainguard Agent Activity Data:
+* Chainguard controller or webhook name
+* "Last seen" timestamp
+
+Cluster Container Images Data:
+* Image name
+* Image hash
+* Type of the workload running this Image (`Deployment`, `ReplicaSet`, `Pods`)
+* Metadata of the workload running this image:
+    * Name
+    * Namespace
+
+Other Cluster Data:
+* Namespace labels, to allow Enforce to recognize when namespace enforcement is enabled or disabled
+* Node metadata
+
+
+## Compliance data
+Chainguard Enforce also collects the following compliance data:
+* Data regarding the packages constituting the image (SBOM), which are:
+    * Package name
+    * Package version
+    * Package PURL
+* Image compliance related to Enforce policies
+
+
+## Data Collection FAQ's
```

Comment:
We have `FAQs` elsewhere (see https://edu.chainguard.dev/open-source/wolfi/faq/)

```suggestion
## Data Collection FAQs
```